CHANGELOG.md

```diff
@@ -1,75 +1,47 @@
 # Changelog
 
-This changelog documents user-facing updates (features, enhancements, fixes, and deprecations) to the `modal` client library. Patch releases are made on every change.
-
-The client library is still in pre-1.0 development, and sometimes breaking changes are necessary. We try to minimize them and publish deprecation warnings / migration guides in advance, typically providing a transition window of several months.
-
-You can also refer to the [1.0 migration guide](https://modal.com/docs/guide/modal-1-0-migration) for a summary of major breaking changes that we are rolling out before releasing version 1.0.
-
-We appreciate your patience while we speedily work towards a stable release of the client.
+This changelog documents user-facing updates (features, enhancements, fixes, and deprecations) to the `modal` client library.
 
 ## Latest
 
 <!-- NEW CONTENT GENERATED BELOW. PLEASE PRESERVE THIS COMMENT. -->
 
-#### 1.0.0.dev18 (2025-05-15)
-
-- Removed `modal.Mount` as a public object along with the various `mount=` parameters where Mounts could be passed into the Modal API. Usage can be replaced with `modal.Image` methods, e.g.:
-  ```python
-  @app.function(image=image, mounts=[modal.Mount.from_local_dir("data", "/root/data")])  # This is now an error!
-  @app.function(image=image.add_local_dir("data", "/root/data"))  # Correct spelling
-  ```
-
+### 1.0.0 (2025-05-16)
 
-#### 1.0.0.dev16 (2025-05-15)
+With this release, we're beginning to enforce the deprecations discussed in the [1.0 migration guide](https://modal.com/docs/guide/modal-1-0-migration). Going forward, we'll include breaking changes for outstanding deprecations in `1.Y.0` releases, so we recommend pinning Modal on a minor version (`modal~=1.0.0`) if you have not addressed the existing warnings. While we'll continue to make improvements to the Modal API, new deprecations will be introduced at a substantially reduced rate, and support windows for older client versions will lengthen.
 
-- Referencing the deprecated `modal.Stub` object will now raise an `AttributeError`, whereas previously it was an alias for `modal.App`. This is a simple name change.
+⚠️ In this release, we've made some breaking changes to Modal's "automounting" behavior.️ If you've not already adapted your source code in response to warnings about automounting, Apps built with 1.0+ will have different files included and may not run as expected:
```

Comment:
Nice emoji

Reply:
It's not our trademark 😵‍💫 but it'll do!